todos-app/backend/task_service/spec/requests/task_spec.rb

```diff
@@ -0,0 +1,176 @@
+require 'rails_helper'
+
+RSpec.describe "Tasks", type: :request do
+  describe "GET /tasks" do
+    context "given no task exists" do
```

Comment:
If possible, If possible, it would be better if you could start a sentence in the context with 'when', 'with' or 'without'.
https://www.betterspecs.org/

Reply:
Good to know. I'll fix my wordings 👍